recipes-kernel/linux/linux-yocto-onl_5.15.bb

```diff
@@ -6,13 +6,13 @@ KCONF_BSP_AUDIT_LEVEL = "1"
 
 LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
-LINUX_VERSION ?= "5.15.44"
+LINUX_VERSION ?= "5.15.56"
 # https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/log/?h=linux-5.15.y
-SRCREV_machine ?= "4e67be407725b1d8b829ed2075987037abec98ec"
+SRCREV_machine ?= "760adb59f6211e157dd587927ac26c42abc81550"
 
 # Use commit for kver matching (or close to) LINUX_VERSION
 # https://git.yoctoproject.org/yocto-kernel-cache/log/kver?h=yocto-5.15
-SRCREV_meta ?= "529199264800b52ae173fc91241c8e64615850e3"
+SRCREV_meta ?= "624cd634ceb095c2949b1cf8b69b0c96b098cb44"
```

Comment:
```suggestion
SRCREV_meta ?= "3b1dc2f1fcd869f97901402759b859035984aa7f"
```

Reply:
Have you tested this?

Reply:
No. I have not started building the image with the old or new SRCREV_meta, but i assumed it might be good to go for the latest and greatest, bumping to 5.15.55 before building and testing. Or is this already tested?

Reply:
If you want to add another round of testinging and verifying we can update this, but then this will delay this PR, because I will first need to check first that there are no unexpected changes in updating the meta SRCREV.

Reply:
I'd suggest leaving it as-is (as these SRCREVs are already tested by me), and bump the meta in a future PR.